packages/teams-js/test/public/call.spec.ts

```diff
@@ -88,7 +88,7 @@ describe('call', () => {
     validateCallDeepLinkPrefix(callDeepLink);
     validateDeepLinkUsers(callDeepLink, mockStartCallParams.targets);
 
-    utils.respondToMessage(executeDeepLinkMsg, false, true);
+    utils.respondToMessage(executeDeepLinkMsg, true);
```

Comment:
I was going to add this, but throwing an error seemed wrong to me since we are resolving a Promise<boolean> to inform if the call succeeded or failed. I thought about it more and elected to add to the resolution of the call.startCall function to now throw a new `errorCallNotStarted` error and added a test case.

Reply:
I agree that it would make a lot more sense if this was a `Promise<void>` function, unfortunately it looks like this will cause breaking issues and so I had to find another solution that accomplished the same expected results. 
`
Good catch on the documentation, I will update it so that we know it will always resolve to `true` or it will `throw` a new `Error`

Interesting that the return value in the docs says that, I will update it to 

```
@returns If the call is successfully initiated

```